src/main/java/com/google/graphgeckos/dashboard/storage/DataRepository.java

```diff
@@ -32,7 +32,7 @@
    * @return true only if the operation completed successfully.
    * @throws IllegalArgumentException if entryData is null
```

Comment:
Is this  `@throws` declaration up-to-date now?
What happens at runtime if this method happens to get called on an instance of `DatastoreRepository` with an `entryData` of `null`? Wouldn't it throw a `NullPointerExpection`, or it throws an `IllegalArgumentException`, in which case this declaration is up-to-date?

Reply:
oops, I forgot to remove it